packages/core/deprecations/core-deprecations-server-internal/src/mocks/index.ts

```diff
@@ -0,0 +1,10 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+export { mockDeprecationsRegistry } from './deprecations_registry.mock';
+export { mockDeprecationsFactory } from './deprecations_factory.mock';
```

Comment:
These two mocks were only used for internal testing. There is no code duplication involved this time.